### DISCUSSION TITLE
continuous 298c31e

### DISCUSSION THREAD

**Discussion by Terracotta-6** [artisan-roaster-scope/artisan#1498] [General]
https://github.com/artisan-roaster-scope/artisan/assets/149906183/f02ede74-cc4b-4c6e-a100-22227de94521

**Comment by Terracotta-6** [comment]
Artisan 2.8.2 (f525b16) The control logic is correct

**Reply by Terracotta-6** [reply]
https://github.com/artisan-roaster-scope/artisan/assets/149906183/9919c2e6-d2d5-4729-9c4a-d8e9c7ebc70d

**Comment by MAKOMO** [comment]
Thanks for your report. My last commit should fix this special case of the PID diaog which occurs if TC4 is the main device, but the TC4 internal PID firmware is not used.

**Reply by Terracotta-6** [reply]
Thanks for you fix .
